src/lib/gadgets/gadgets.ts

```diff
@@ -104,4 +104,33 @@ const Gadgets = {
   xor(a: Field, b: Field, length: number) {
     return xor(a, b, length);
   },
+
+  /**
+   * Bitwise NOT gate on {@link Field} elements. Equivalent to the [bitwise
+   * NOT `~` operator in JavaScript](https://developer.mozilla.org/en-US/docs/
```

Comment:
we should add a small disclaimer here that it behaves slightly different than the JavaScript operator - see the bindings PR for an explanation